changelogs/fragments/494-LDAP-security-profile-task.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - sonic_ldap - Add prerequisite task for LDAP security profile (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/494).
```

Because this fix is confined to our internally used tests, it is not visible to customers, so the changelog classification should be "trivial" to prevent this change from appearing in the release note.

I am also suggesting a minor change in the wording to clarify that it is a "test" change.

```suggestion
trivial:
  - sonic_ldap - Add prerequisite task for LDAP security profile tests (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/494).
```